assessment.js

```diff
@@ -1,8 +1,8 @@
 //We'll learn about require later in the course
 var ask = require('./ask.js');
 
-var answer = 'not empty';
+//var answer = 'not empty';
 
-while (answer !== '' && answer !== 'SeCrEt') {
-  answer = ask("Guess my secret? ");
+while (var i <= 10) {
+  ask("Guess my secret? ");
```

Comment:
Not quite - when does the player's response get used?
We were thinking of something more like this:
```javascript
var ask = require('./ask.js');
var answer = 'not empty';
while (answer !== '' && answer !== '3') { // secret number is 3
  answer = ask("Guess a number between 1 and 10.")
}
```